azure.sh

```diff
@@ -29,7 +29,7 @@ az login --service-principal --username "${AZURE_CLIENT_ID}" --password "${AZURE
 #---------------------------------------------------------------
 
 # List all resources from AZURE_RESOURCE_GROUP
```

Comment:
You should probably also change the comment and there is one more use of this variable on line 66.

Reply:
oups, friday... :S